server/middleware/caseloadCheckMiddleware.ts

```diff
@@ -1,12 +1,12 @@
 import { RequestHandler } from 'express'
 import logger from '../../logger'
 
-export default function caseloadCheckMiddleware(enabledPrisons: string): RequestHandler {
+export default function caseloadCheckMiddleware(enabledPrisons: string[]): RequestHandler {
   return (req, res, next) => {
-    if (res.locals && res.locals.user && enabledPrisons.split(',').includes(res.locals.user.activeCaseLoadId)) {
+    if (enabledPrisons.includes(res.locals.user?.activeCaseLoadId)) {
       return next()
     }
-    logger.error('User is not authorised to access this')
+    logger.warn('User is not authorised to access this')
```

Comment:
warn rather than error - this is not an unexpected error that we necessarily need to investigate.